src/api/types/DexDtos.ts

```diff
@@ -1109,6 +1131,62 @@ export class CreatePoolResDto extends ChainCallDTO {
   }
 }
 
+export class TransferUnclaimedFundsDto extends SubmitCallDTO {
+  @IsNotEmpty()
+  @ValidateNested()
+  @Type(() => TokenClassKey)
+  public token0: TokenClassKey;
+
+  @IsNotEmpty()
+  @ValidateNested()
+  @Type(() => TokenClassKey)
+  public token1: TokenClassKey;
+
+  @EnumProperty(DexFeePercentageTypes)
+  public fee: DexFeePercentageTypes;
+
+  @IsUserAlias()
+  @IsNotEmpty()
+  public secureWallet: UserAlias;
```

Comment:
```suggestion
  @IsUserRef()
  @IsNotEmpty()
  public secureWallet: UserRef;
```

Typically we will use the `UserAlias` type for values stored on chain (i.e. classes that extend `ChainObject`) and `UserRef` for values provided in DTOs (e.g. classes that extend `SubmitCallDTO` or `ChainCallDTO`).

For reference:

https://github.com/GalaChain/sdk/blob/main/BREAKING_CHANGES.md

```
Starting from version 2.0.0 we enforce strong validation for user related fields in the DTOs and chain objects. There are two types of user related fields:

    User alias (UserAlias): A unique identifier for each user that is stored in objects saved on chain and processed within the majority of methods. Each chain user has a single unique alias (like client|abc or eth|<checksumed-addr>).
    User reference (UserRef): A reference to the user object, used in the DTOs. A user reference unambiguously identifies the user, but a single user can have multiple references (for instance client|abc, checksumed eth address, and lower-cased eth address can all refer to the same user).

```